### PR TITLE
fix: Increase timeout

### PR DESCRIPTION
**Pull request by doosuu**
Increase timeout for app container builds to allow C++ builds to take longer.

### CODE REVIEW

File: runtime_kanto/test/integration/integration_test.py [hunk 1]
```diff
@@ -57,13 +58,16 @@ def check_container_is_running(container_name: str) -> bool:
         app_name (str): App name
     """
     return json.loads(
-        subprocess.run(
-            ["kanto-cm", "get", "-n", container_name], stdout=subprocess.PIPE
-        ).stdout.decode("utf-8")
+        subprocess.check_output(
```
Comment (MP91) [inline runtime_kanto/test/integration/integration_test.py]: Any reason to change this? subprocess.run is the recommended way to do it.

File: runtime_kanto/test/integration/integration_test.py [hunk 2]
```diff
@@ -18,18 +18,19 @@
 import sys
 import time
 from pathlib import Path
-from re import Pattern, compile
+from re import Pattern
+from re import compile as re_compile
```
Comment (MP91) [inline runtime_kanto/test/integration/integration_test.py:22]: is there any benefit?

Comment (doosuu) [reply runtime_kanto/test/integration/integration_test.py:22]: `compile` is a default keyword which is shadowed by this import, hence the alias. My local linter was complaining about it.

File: runtime_kanto/test/integration/integration_test.py (hunk 1 above)
Comment (doosuu) [reply runtime_kanto/test/integration/integration_test.py]: Reverted.

**Review by MP91 — APPROVED** [review]
lgtm

**Pull request merged**